packages/experimental/Shimmer/src/ShimmerTokens.ts

```diff
@@ -0,0 +1,15 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { ShimmerTokens } from './Shimmer.types';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { getCurrentAppearance } from '@fluentui-react-native/theming-utils';
+
+export const defaultShimmerTokens: TokenSettings<ShimmerTokens, Theme> = (theme: Theme) =>
+  ({
+    angle: 0,
+    delay: 0,
+    duration: 7000,
+    shimmerColor: getCurrentAppearance(theme.host.appearance, 'light') === 'light' ? '#E1E1E1' : '#404040',
```

Comment:
Can the shimmer colors be tokenized or use the theme object?

Reply:
+1, would be nice if these used global colors instead of hex colors, at minimum.

Reply:
I have just moved the values that were already present in Shimmer.styling.ts file into this separate token file. I have not added anything over the existing code here.

Reply:
Please do let me know if I should replace these with something else, I am not sure what would go here. These hex values / getCurrentAppearance logic are not used in Android

Reply:
I just checked, and they actually don't exist as global tokens? Where did get these specific colors from?

Reply:
Okay, probably not @ayush547 's current problem if they weren't global tokens in the first place.

Reply:
Alright, leaving these as such. 
FYI, Shimmer tokens on windows uses theme obj but the wave color alone is still a hex value, not sure if that should be replaced with some global token or something from the theme obj.